### ISSUE TITLE
Essentials Chat requesting one single chat or a way to turn off local chat or have global chat be the main chat.

### ISSUE THREAD

**Issue by AngelicBeatz** [ProjectEssentials/ProjectEssentials-Chat#13]
### Is your feature request related to a problem? Please describe.
> - A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

### Describe the solution you'd like
> - A clear and concise description of what you want to happen.
I want to be able to keep the global chat and get rid of the local chat. typing the ! before the message is an extra set that is unneeded.
### Describe alternatives you've considered
> - A clear and concise description of any alternative solutions or features you've considered.
I tried turning off the ranged chat as you have it in the Configuration section with the blue text boxes, but that disables the chat entirely and I want the chat to be there. Is there a way to just have global chat? or a way to turn off local chat?
### Additional context
> - Add any other context or screenshots about the feature request here.

**Comment by MairwunNx** [comment]
Have a nice evening!, @AngelicBeatz 

This feature already implemented, set `enableRangedChat` to `false`.

If ranged chat disabled, will used by default common chat.

**Comment by MairwunNx** [comment]
If you disable ranged chat then as default chat template will use `commonPattern`, or something like that. 

If I'm not understanding you right, please give me more info. Thanks!

**Comment by AngelicBeatz** [comment]
for some reason when I disable the ranged chat it still lets me curse in chat.

**Comment by AngelicBeatz** [comment]
do you have a discord that I could join for this project, also do you know off the top of your head what the permission for /back is? I would have thought it was esb.back

**Comment by MairwunNx** [comment]
> for some reason when I disable the ranged chat it still lets me curse in chat.

Sorry, curse?

**Comment by MairwunNx** [comment]
> do you have a discord that I could join for this project, also do you know off the top of your head what the permission for /back is? I would have thought it was esb.back

Yes, we have discord chat, you can find link to this in Core or Permissions repositories.

Permission to back confused many people, at now is `ess.teleport.back`, and in 2.0.0 it will changed to `ess.back`.

**Comment by AngelicBeatz** [comment]
cus/bad language

**Comment by MairwunNx** [comment]
Can you re-describe this issue now?  This problem can be reproduced?
